clang/test/Sema/free_function_attribute_check.cpp

```diff
@@ -0,0 +1,59 @@
+//==---- free_function_attribute_check.cpp ---------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: %clang_cc1 -fsycl-is-device -triple -spir64-unknown-unknown -verify %s
+
+// expected-no-diagnostics
+
+// This test checks that non-constexpr values appearing in
```

Comment:
Can you please update this comment?

You can say something along the lines of "This test ensures that the compiler does not crash when functions or methods with add_ir_attributes_function attribute in dependent contexts are checked for the presence of the free function property"

Reply:
OK, I've updated the comment.